dev/io.openliberty.mcp.internal/test/io/openliberty/mcp/internal/test/AcceptHeader.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.mcp.internal.test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import io.openliberty.mcp.internal.HeaderValidation;
+
+public class AcceptHeader {
+    @Test
+    public void testAcceptHeaderExactlyRequired() {
+        assertTrue(HeaderValidation.acceptContains("application/json, text/event-stream", "application/json"));
+        assertTrue(HeaderValidation.acceptContains("application/json, text/event-stream", "text/event-stream"));
+    }
+
+    @Test
+    public void testAcceptHeaderWildcard() {
+        assertFalse(HeaderValidation.acceptContains("*/*", "application/json"));
+        assertTrue(HeaderValidation.acceptContains("*/*", "text/event-stream"));
+    }
+
+    @Test
+    public void testAcceptHeaderWithQuality() {
+        String mime = "application/json;q=1.0, text/*;q=0.5";
+        assertTrue(HeaderValidation.acceptContains("application/json", mime));
+        assertTrue(HeaderValidation.acceptContains("text/event-stream", mime));
+    }
```

Comment:
```suggestion
        assertTrue(HeaderValidation.acceptContains("application/json", mime));
        assertTrue(HeaderValidation.acceptContains("text/event-stream", mime));
        assertTrue(HeaderValidation.acceptContains("text/foo", mime));
        assertFalse(HeaderValidation.acceptContains("application/yaml", mime));
        assertFalse(HeaderValidation.acceptContains("image/png", mime));
    }
```

Some negative tests would be good here (or you could add them in another test method)

Reply:
**testAcceptHeaderWithQuality**  this test is meant to assertFalse for both test as header is too strict and doesn't match